nasa-gibs/worldview
# WV-3106: AERONET descriptions

## Description

Fixes #WV-3106.

Added layer descriptions; renamed layer titles, added image previews; disabled snapshots for the 4 layers. 

## How To Test

1. Open with [these URL parameters](http://localhost:3000/?z=4&ics=true&ici=5&icd=60&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,DAILY_AERONET_ANGSTROM_440-870NM,DAILY_AERONET_AOD_500NM,AERONET_ANGSTROM_440-870NM,AERONET_AOD_500NM,VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2024-04-29-T13%3A08%3A59Z)
2. Check layer descriptions, image previews, and that you are not able to take image snapshots with AERONET layers loaded. 



@nasa-gibs/worldview
